.changeset/plenty-dolls-add.md

```diff
@@ -0,0 +1,60 @@
+---
+'@backstage/cli': major
+'@backstage/create-app': major
```

Comment:
This should be a minor change to any of the packages that read the env var directly, and a patch change for create-app. I doesn't really touch the cli or e2e-test either, since it's just dependency updates that will be reflected anyway.

Reply:
Thank you so much. It directly affects only ```techdocs-backend```.